keps/sig-release/3027-slsa-compliance/README.md

```diff
@@ -0,0 +1,894 @@
+-->
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+### Risks and Mitigations
+
+Failure to pursue this enhancement poses three problems for the project:
+
+
+#### Leaving Common Threats Unaddressed
+
+SLSA requirements call attention to the most vulnerable areas of Software 
+Supply Chains. Failure to pursue this enhancement leaves some of those areas 
+exposed.
+
+#### Failure to Provide Certainty Downstream
+
+As projects up and down the stream harden their supply chains, Kubernetes
+cannot afford to be the weakest link. Our base images, the [Distroless Images are SLSA 
+2 compliant now](https://security.googleblog.com/2021/09/distroless-builds-are-now-slsa-2.html),
+for example. If we want to be good team players, our turn is up.
+
+
+<!--
+What are the risks of this proposal, and how do we mitigate? Think broadly.
+For example, consider both security and how this will impact the larger
+Kubernetes ecosystem.
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+
+Consider including folks who also work outside the SIG or subproject.
+-->
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include API specs (though not always
+required) or even code snippets. If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+<!--
+
+### Test Plan
+
+**Note:** *Not required until targeted at a release.*
+
+Consider the following in developing a test plan for this enhancement:
+- Will there be e2e and integration tests, in addition to unit tests?
+- How will it be tested in isolation vs with other components?
+
+No need to outline all of the test cases, just the general strategy. Anything
+that would count as tricky in the implementation, and anything particularly
+challenging to test, should be called out.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
+when drafting this test plan.
+
+[testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
+-->
+
+### Graduation Criteria
+
+This KEP can be considered complete when one of two scenarios is reached:
+
+1. All SLSA levels have been successfully complied with 
+1. The community determines one of the levels as not implementable. This
+may be concluded if the nature of necessary changes proves to be too
+disruptive or implies altering aspects of technical environments
+and/or systems, contributor experience, policy, and other domains beyond what
+deems to be acceptable.
+
+Tracking issues will be opened to track and discuss the viability of the
+required enhancements to reach each SLSA level.
```

Comment:
Let's add that we also update the roadmap to reflect the current state.

Reply:
OK, added.